lib/oop.js

```diff
@@ -23,7 +23,36 @@ const mixin = (target, source) => {
   Object.assign(target, mix);
 };
 
+const ASCII_A = 65;
+
+// Convert instance with public fields to instance with private fields
+//   instance - <Object>, source instance
+// Returns: <Object> - destination instance
+//
+// Example: common.privatize({ private: 5, f() { return this.private; } });
+const privatize = instance => {
+  const iface = {};
+  const fields = Object.keys(instance);
+  for (const fieldName of fields) {
+    const field = instance[fieldName];
+    if (typeof field === 'function') {
+      const bindedMethod = field.bind(instance);
+      iface[fieldName] = bindedMethod;
+    } else if (fieldName === fieldName.toUpperCase()) {
```

Comment:
Nested `if` doesn't cries about "let's keep constants". What if 
I guess the loop body could be refactored to:

```js
const field = instance[fieldName];
const isMethod = typeof field === 'function';
if (!isMethod && !isConstantName(fieldName)) continue; // skipping private fields
iface[fieldName] = isMethod ? field.bind(instance) : field;
```